lib/tokenable/revokeable.rb

```diff
@@ -0,0 +1,33 @@
+module Tokenable
+  module Revokable
+    extend ActiveSupport::Concern
+
+    def token_revoked?(revoke_token)
+      !revoke_token.in?(revoke_tokens_hash.keys)
+    end
+
+    def issue_revoke_token
+      issued_token = SecureRandom.hex
+      self.update(
+        revoke_tokens_key => revoke_tokens_hash.merge(
```

Comment:
We probably don't want to add a token every time, thish would grow huge and become a performace bottleneck in rails and the DB.

We should instead pick the most recent one each time a new jwt token is issued. 99% of the time, there will be a single token in use. The second one comes into play when you cycle the tokens but want to keep existing clients working while you update them. Think like aws credentials, they only let you have two as they're speficially designed for cycling every X days.

Reply:
This is what makes me think we should just go with a single "secret" for now. It removes a lot of complexity. We would just keep it the same until it's under.